packages/react/src/components/ChatInput/ChatInput.js

```diff
@@ -137,9 +144,13 @@ const ChatInput = ({ scrollToBottom }) => {
   };
 
   const sendMessage = async () => {
+    // clearQuotedMessages();
     scrollToBottom();
     messageRef.current.style.height = '44px';
     const message = messageRef.current.value.trim();
+    console.log(message);
```

Comment:
```suggestion
```

Reply:
Yes, sure. I will clean up the code in the final PR (whenever needed), I just wanted to confirm if I should work further on this, that is why I created this one.